### PR TITLE
Staging2

### PR DESCRIPTION
**Pull request by navkelley**
Refactored to create objects per assignment

### CODE REVIEW

File: bartender.js [hunk 1]
```diff
@@ -1,24 +1,35 @@
...
-  var pantry = {
-    strong: ["Glum of rum", "slug of whisky", "splash of gin"],
-    salty: ["Olive on a stick", "salt-dusted rim", "rasher of bacon"],
-    bitter: ["Shake of bitters", "Splash of tonic", "twist of lemon peel"],
-    sweet: ["Sugar cube", "Spoonful of honey", "Splash of cola"],
-    fruit: ["Slice of orange", "Dash of cassis", "Cherry on top"],
-  }
+  //ingredients constructor 
+  var Ingredients = function (ingredients) {
+    this.ingredients = ingredients
```
Comment (gianpaj) [inline bartender.js]: don't forget the semicolons (`;`) in here and some other lines

File: bartender.js [hunk 2]
```diff
@@ -1,24 +1,35 @@
...
   orderValue = [];
 
   //constructor for ingredients 
-  var userOrder = function (strong, salty, bitter, sweet, fruit) {
-    this.strong = orderValue[0];
-    this.salty = orderValue[1];
-    this.bitter = orderValue[2];
-    this.sweet = orderValue[3]; 
-    this.fruit = orderValue[4];
+  var UserOrder = function (strongItem, saltyItem, bitterItem, sweetItem, fruityItem) {
```
Comment (gianpaj) [inline bartender.js]: While you're not using this function/Object just yet, notice that the arguments (strongItem, etc) will not be used. `this.strongItem` is different than the argument `strongItem`.

Probably seeing the Object being used might help understand it better.

Comment (navkelley) [inline bartender.js]: I am still trying to understand objects and how to best use them multiple times. I will go over this with my mentor, thank you for pointing out that those are not the same! Now I can get clarification!

**Comment by gianpaj** [comment]
No problem. In general, when you do a pull-request and reviewer mentions few issues, if those make sense, you should commit onto the same branch (in this case `staging2`) so the merge go to master with the applied fixes.

**Pull request merged**